pkg/config/ovn/ovn.go

```diff
@@ -30,20 +30,10 @@ const (
 )
 
 type OVNKubernetesConfig struct {
-	// Configuration for microshift-ovs-init.service
-	OVSInit OVSInitConfig `json:"ovsInit,omitempty"`
 	// MTU to use for the pod interface. Default is 1500.
 	MTU int `json:"mtu,omitempty"`
 }
 
-type OVSInitConfig struct {
-	// disable microshift-ovs-init.service.
-	// OVS bridge "br-ex" needs to be configured manually when disableOVSInit is true.
-	DisableOVSInit bool `json:"disableOVSInit,omitempty"`
-	// Uplink interface for OVS bridge "br-ex"
-	GatewayInterface string `json:"gatewayInterface,omitempty"`
-}
-
 func (o *OVNKubernetesConfig) Validate() error {
 	// br-ex is required to run ovn-kubernetes
 	err := o.validateOVSBridge()
```

Comment:
This validation would fail if br-ex doesn't exist, maybe the validation shall be removed given br-ex is created by ovnk.

Reply:
I think that is what we expect. `br-ex` shall be created by `microshift-ovs-init.service` before microshft starts.

Reply:
ack+. I mis-read the commit message as the br-ex will be created by ovnkube-node.